src/data/faq_de.json

```diff
@@ -44,1240 +44,1242 @@
+                            "Die Corona-Warn-App fragt Ihren Standort nicht ab und hat auch keine Berechtigung zur Standortabfrage. Die Meldung geht auf eine Besonderheit von Android zurück: Bluetooth-Geräte in Ihrer Nähe können grundsätzlich nur gefunden werden, wenn die Standortverwendung allgemein auf Ihrem Gerät aktiviert ist (siehe <a href='https://support.google.com/android/answer/9888358?hl=de' target='_blank' rel='noopener noreferrer'>COVID-19-Benachrichtigungen auf Ihrem Android-Smartphone aktivieren</a> und <a href='https://support.google.com/android/answer/9930236?hl=de' target='_blank' rel='noopener noreferrer'>COVID-19-Benachrichtigungssystem und Android-Standorteinstellungen</a> in der Android-Hilfe sowie <a href='https://developer.android.com/guide/topics/connectivity/bluetooth' target='_blank' rel='noopener noreferrer'>Bluetooth Overview</a> in der Android-Entwicklerdokumentation). Das bedeutet aber nicht, dass Apps, die Bluetooth nutzen, auch automatisch Ihren Standort erfassen.",
+                            "Da die Corona-Warn-App Geräte in Ihrer Nähe über Bluetooth erkennt, muss die allgemeine Systemeinstellung 'Standort verwenden' als Voraussetzung aktiviert sein. Allerdings wird die App zu keiner Zeit Ihren Standort erfassen oder GPS verwenden. Sie können das gerne selbst folgendermaßen nachprüfen:",
+                            "<ol><li>Öffnen Sie die Einstellungen Ihres Smartphones (nicht die Einstellungen in der App!).</li><li>Wählen Sie 'Sicherheit & Standort' > 'Standort' > 'Berechtigungen auf App-Ebene'.</li><li>Sehen Sie sich die Liste der Apps an, die auf Ihren Standort zugreifen können, wenn Sie es zulassen. Sie sehen: Die Corona-Warn-App ist nicht aufgeführt.</li></ol>",
+                            "An dieser Stelle könnten Sie anderen Apps die Berechtigung für den Standortzugriff erteilen oder entziehen, aber da die Corona-Warn-App diese Berechtigung nicht braucht, ist sie auch nicht aufgeführt.",
+                            "Hinweis: Wie oben beschrieben: Aktivieren Sie die allgemeine Systemeinstellung 'Standort verwenden' unter 'Einstellungen' > 'Sicherheit & Standort' > 'Standort', da sonst der Austausch der verschlüsselten Zufalls-IDs zur Risiko-Ermittlung nicht funktioniert."
+                        ]
+                    },
+                    {
+                        "title": "Beim Einscannen des QR Codes erhalte ich einen Fehler. Was kann ich tun?",
+                        "anchor": "qr_scan",
+                        "active": true,
+                        "textblock": [
+                            "Dieses Problem kann zwei verschiedene Ursachen haben:",
+                            "<b>Der QR Code wurde bereits einmal zur Registrierung benutzt</b>",

```

Comment:
In the line above this one
  "Dieses Problem kann zwei verschiedene Ursachen haben:",

"zwei" needs to be changed to "drei"

Reply:
ok